std_msgs/Header header
grid_map_msgs/GridMapInfo info
GridLayer[] layers
